lib/types/operation.js

```diff
@@ -90,7 +90,7 @@ var Operation = module.exports = function (parent, scheme, operationId, httpMeth
     if (innerType && innerType.toString().toLowerCase() === 'boolean') {
       param.allowableValues = {};
       param.isList = true;
-      param['enum'] = ['true', 'false'];
+      param['enum'] = [true, false]; // use actual primitives
```

Comment:
We probably need to handle both or update the converter to do type coercion.  In Swagger 1.x, boolean values were their string counterpart so this change works great for Swagger 2.0 but will likely fail for Swagger 1.x.